src/ansys/fluent/core/post_objects/post_objects_container.py

```diff
@@ -84,10 +84,29 @@ def _init_module(self, obj, mod, post_api_helper):
             if cls.__class__.__name__ in (
                 "PyLocalNamedObjectMetaAbstract",
             ) and not inspect.isabstract(cls):
+                cont = PyLocalContainer(self, cls, post_api_helper, cls.PLURAL)
+
+                def _add_create(py_cont):
+                    def _create(**kwargs):
+                        new_object = py_cont.__getitem__(
+                            py_cont._get_unique_chid_name()
+                        )
+                        unexpected_args = list(set(kwargs) - set(new_object()))
```

Comment:
Creating a list is unnecessary. We can just call `next(iter(...))` when constructing the `TypeError`.